apps/prairielearn/src/tests/courseEditor.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable perfectionist/sort-sets */
```

Comment:
The `files` in this test seem ordered in a more natural way, e.g. README / infoCourse.json first.